pipe/section/section_impls/tagging_transformer/src/lib.rs

```diff
@@ -85,7 +85,7 @@ where
                                                 .iter()
                                                 .map(std::borrow::ToOwned::to_owned)
                                                 .collect::<Vec<_>>();
-                                            let tag = Arc::new(StringArray::from(vec![self.text.clone()]));
+                                            let tag = Arc::new(StringArray::from(vec![self.text.clone(); values[0].len()]));
```

Comment:
I don't understand this solution, why are so many duplicates of `self.text` needed?

Reply:
The transformer worked perfectly fine when there was only one row being sent in the record batch. But when there are multiple rows, each one needs to have the value added. The error was arising from the last column, the one with `self.text`, only having one value when the other columns each had more. 

The way I tested this was with an excel source -> tagging transformer -> hello world destination, plus adding logging around these lines to see what `values` was before and after pushing `tag` to it.

Reply:
Thank you, that makes sense